pnpm-lock.yaml

```diff
@@ -83,7 +83,7 @@ importers:
       create-vite: latest
       kolorist: ^1.5.0
     dependencies:
-      create-vite: link:../create-vite
```

Comment:
Should this file have changed?

Reply:
These lines were changed automatically as a result of running `pnpm install`.

Reply:
I wonder if that's intended though? I wonder if there's a frozen lock file equivalent to what we use for yarn. Probably worth reverting the change!